tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
I'm still not sure about the question in my previous comment, but I think it probably doesn't matter since it seems to be an edge case.

No further comments from the peanut gallery for now.